templates/lx_userdata.sh

```diff
@@ -208,7 +208,7 @@ finally() {
   open-ssh
   publish-artifacts
   # shellcheck disable=SC2154
-  if [ "$build_type" == "$build_type_standalone" ] && [ "${wam_version}" != "" ]; then
+  if [ "$build_type" == "$build_type_standalone" ] && [ "${scan_slug}" != "" ]; then
```

Comment:
This is _always_ set at the moment. Even if the user blanks `var.scan_s3_bucket` and `var.scan_s3_prefix`, it will end up as `/`...

Reply:
🤦 Let me fix...